index.js

```diff
@@ -0,0 +1,50 @@
+function validatePassword(valid) {
+  if (passwordLength(valid) === true && passwordLowerCase(valid) === true && passwordUpperCase(valid) === true && passwordNumericValue(valid) === true && passwordSpecialCharacter(valid) === true) {
```

Comment:
`if (passwordLength(valid) === true)` is exactly the same as `if (passwordLength(valid))` the `=== true` is redundant.